CHANGELOG.md

```diff
@@ -2,7 +2,7 @@ To Be Released
 ===============
 
 * New: FontAwesome updated to version 4.7.0
-* New: `Photos` plugin support for photo gallery creation
+* New: `Photos` plugin support for photo gallery creation and lightbox
```

Comment:
I left a comment earlier. I don't know why github removed it. I said

Release note should read

```
New: `Photos` plugin support for photo gallery creation
New: Lightbox support for `Photos` plugin
```

This will highlight your contribution more.

Reply:
done in next push